.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
```

```suggestion
blank_issues_enabled: true
```
let the option we do nudge user already ...